### DISCUSSION TITLE
Installer is user friendly to set up but, can we do with mannual configuration?

### DISCUSSION THREAD

**Discussion by vivek4321** [AquilaCMS/AquilaCMS#210] [Q&A] [answered]
Installer is user-friendly to set up but, can we do it with mannual configuration?

I prefer .env or .yml way so we can keep these efforts of maintenance installer on other way.

**Comment by devaquila** [comment, marked as answer]
You are a lucky man!
We implemented this feature a few days ago :)
https://github.com/AquilaCMS/AquilaCMS/commit/596013182fcf314d0969772bf84cce268bf631b0
https://github.com/AquilaCMS/AquilaCMS/commit/e67a6a4294c6c3efaf591ad3a16c3b1f0a0f3224

Please, feel free to give your feedback on this
